tools/cdi-func-test-file-host-init/main.go

```diff
@@ -52,6 +52,8 @@ func main() {
 		[]string{".qcow2", ".gz"},
 		[]string{".qcow2", ".xz"},
 		[]string{".qcow2", ".zst"},
+		[]string{".tar", ".gz"},
```

Comment:
I wondering, if it can deal with file like xxx.raw.tar.gz ?

Reply:
@a180285  Sure, actually we match file by magic number instead of extension.